src/app/messages/pl/messages.json

```diff
@@ -107,7 +107,9 @@
   "i18n:notification:update:body": "Kliknij poniżej, by przejść do najnowszej wersji oprogramowania.",
   "i18n:notification:update:header": "Dostępna aktualizacja",
   "i18n:notification:update:reload": "Aktualizuj",
+  "i18n:notification:toast:archive:load-failure": "Nie możemy załadować tej strony. Sprawdź połączenie z internetem i spróbuj ponownie.",
   "i18n:notification:toast:search:highlight-not-found": "Nie odnajdujemy wyników dla tego zapytania. Spróbuj innego.",
+  "i18n:notification:toast:search:load-failure": "Nie możemy zakończyć wyszukiwania. Sprawdź połączenie z internetem i spróbuj ponownie.",
```

Comment:
@ewaforman Would you mind taking a look at these translations for "We're unable to load this page" and "We're unable to complete your search request"? I'd be happy to make any changes you suggest!

Reply:
@jivey Looks good.